IdentityService/DataAccessLayer/Data/UserDbContext.cs

```diff
@@ -0,0 +1,22 @@
+﻿using DataAccessLayer.Models;
+using DataAccessLayer.Repositories;
+using Microsoft.EntityFrameworkCore;
+
+namespace DataAccessLayer.Data
+{
+    public class UserDbContext : DbContext
+    {
+        public UserDbContext(DbContextOptions<UserDbContext> opt) : base(opt) { }
+
+        public DbSet<Client> Clients { get; set; }
+
+        public DbSet<Driver> Drivers { get; set; }
```

Comment:
You have drivers, but i don't see service snd repository for drivers

Reply:
i will add it today